### DISCUSSION TITLE
Ideas to improve script running workflow

### DISCUSSION THREAD

**Discussion by miketrump** [tteck/Proxmox#3773] [Ideas]
Hello
When you encounter any issue running any tteck script that means you will need to start all over again !!
For example: duplicate container number, slight internet disconnection, .....
What I hope is :
1. Ability to edit/rerun last step
2. Ability to resume your interrupted script to continue from where it was stopped whenever that was possible.
3. Ability to easily provide a template/config file for user values (for things like storage place, apt cacher, dns server, etc...) .
When you need to create multiple containers, or recreate a container multiple times to test/fix something, you would like to find a way to pass these values automatically.

Thanks

**Comment by Mellowlynx** [comment]
1/2 probably not going to happen, it's easier to remove the LXC and run the script again.
3. already asked, and his answer was "no" sadly.
